appenders/python/pulse/version.py

```diff
@@ -0,0 +1,15 @@
+#   _______       __
+#  |   _   .--.--|  .-----.-----.
+#  |.  1   |  |  |  |__ --|  -__|
+#  |.  ____|_____|__|_____|_____|
+#  |:  |
+#  |::.|
+#  `---'
+
+__title__ = 'pulse-logging'
+__description__ = 'Pulse Logging'
+__url__ = 'https://pulse-logging.readthedocs.io/en/latest/'
+__version__ = '2.3.1'
+__license__ = 'Apache 2.0'
+__author__ = "phData, Inc."
+__author_email__ = "jordan@phdata.io"
```

Comment:
@afoerster do you have some generic pulse email i could put here?

Reply:
I don't. I think we need to create something like 'open-source@phdata.io' for stuff like this.